index.njk

```diff
@@ -6,33 +6,57 @@
     <title>Towtruck</title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="styles/styles.css" />
+    <script src="https://cdn.tailwindcss.com"></script>
   </head>
-  <body>
-    <h1>Towtruck</h1>
+  <body class="bg-stone-100 text-stone-800">
+    <header class="bg-white">
+      <nav class="w-full flex p-6 border-b-4 border-stone-200">
+        <h1 class="text-3xl">Towtruck</h1>
+      </nav>
+    </header>
 
-    {% for installation in installations %}
-    <table>
-      <caption>
-        {{ installation.name }}'s repos
-      </caption>
-      <thead>
-        <tr>
-          <th scope="col">Name</th>
-        </tr>
-      </thead>
-      <tbody>
-        {% for repo in installation.repos %}
-        <tr>
-          <td>{{ repo.name }}</td>
-        </tr>
-        {% else %}
-        <tr>
-          <td>No repos found</td>
-        </tr>
-        {% endfor %}
-      </tbody>
-    </table>
-    {% endfor %}
+    <div class="container mx-auto flex flex-col mt-6 space-y-6">
+      {% for installation in installations %}
+      <div class="bg-stone-50 overflow-hidden rounded-lg border-b-4 border-stone-200 space-y-4">
+        <h2 class="bg-white border-b-2 border-stone-100 text-xl font-bold p-4">{{ installation.name }}</h2>
+        <p class="mx-4">
+          There
```

Comment:
``` 
{% if installation.repos.length === 1 %}
There is <span class="font-bold">1</span> repository
{% else %} 
There are <span class="font-bold">{{ installation.repos.length }}</span> repositories
{% endif %}
that Towtruck is tracking for <span class="font-bold">{{ installation.name }}</span>.
``` 
Would be clearer here. I think readability is more important than DRY-ness when templating.

Reply:
I've made this change.